charts/kong/ci/__snapshots__/service-proxy-nameoverride-values.snap

```diff
@@ -0,0 +1,1045 @@
+# chartsnap: snapshot_version=v3
+---
+# Source: kong/templates/service-account.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: chartsnap-kong
+  namespace: jx
```

Comment:
Please run the `make test.golden.update` with the kubectl context set to `default`.

Reply:
@pmalek Sorry. Didn't realize needed to run that. Done and updated.